Wobble/Graphics/Sprites/SpriteAlphaMaskBlend.cs

```diff
@@ -0,0 +1,46 @@
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using System;
+
+namespace Wobble.Graphics.Sprites
+{
+    public class SpriteAlphaMaskBlend
+    {
+        static private Game game => GameBase.Game;
+
+        static private readonly BlendState blend = new BlendState
+        {
+            AlphaSourceBlend = Blend.DestinationAlpha,
+            AlphaBlendFunction = BlendFunction.Subtract,
+            AlphaDestinationBlend = Blend.InverseDestinationAlpha
+        };
+
+        static public Texture2D PerformBlend(Texture2D srcTexture, Texture2D srcMask)
+        {
+            var renderTarget = new RenderTarget2D(GameBase.Game.GraphicsDevice, srcTexture.Width, srcTexture.Height, false,
+                GameBase.Game.GraphicsDevice.PresentationParameters.BackBufferFormat, DepthFormat.None);
```

Comment:
There is currently a memory leak here because there's nowhere to dispose the newly created RenderTarget2D (it doesn't exist here nor in the Quaver-side PR). 

You're creating a new RenderTarget2D every single time the user enters the results screen but not caching or disposing it anywhere.

Whether it's here or Quaver-side, this RenderTarget should either be reused or disposed when no longer needed.